inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/instrumentation/context/InspectitContext.java

```diff
@@ -211,12 +211,10 @@ public static InspectitContext createFromCurrent(CommonTagsManager commonTagsMan
      */
     @Override
     public void makeActive() {
-
-        boolean anyDownPropagatedDataOverwritten = dataOverwrites.keySet().stream()
-                .anyMatch(propagation::isPropagatedDownWithinJVM);
+        boolean anyDownPropagatedDataOverwritten = anyDownPropagated();
 
         //only copy if any down-propagating value has been written
-        if (anyDownPropagatedDataOverwritten) {
+        if (anyDownPropagated()) {
```

Comment:
you are not using the boolean variable here which you declared above

Reply:
:+1: great catch..